ext/uri/uriparser/src/UriConfig.h.in

```diff
@@ -37,15 +37,11 @@
  */
 
 #if !defined(URI_CONFIG_H)
```

Comment:
`ext/uri/uriparser/src/UriConfig.h.in` should be deleted as it is stored as `ext/uri/uriparser/src/UriConfig.h`

I found this when coding the download script for #19802.